### PR TITLE
Extracted GameFormatterInterface and implemented 2 example formatters

### PR DESCRIPTION
**Pull request by mannion007**

### CODE REVIEW

File: src/Game/GameFormatterInterface.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?php namespace JoeStrong\RetroAchievements\Game;
+
+interface GameFormatterInterface
+{
+    /**
+     * @param Game[] $games
+     * @return string
+     */
+    public function format(array $games) : string;
```
Comment (tarlepp) [inline src/Game/GameFormatterInterface.php:9]: Style differs from actual implementation `public function format(array $games) : string` vs `public function format(array $games): string`

File: src/Game/Game.php [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<?php namespace JoeStrong\RetroAchievements;
+<?php namespace JoeStrong\RetroAchievements\Game;
```
Comment (tarlepp) [inline src/Game/Game.php:1]: PSR rules says that namespace is next line after `<?php`

File: src/Game/Formatter/HtmlGameFormatter.php [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<?php namespace JoeStrong\RetroAchievements\Game\Formatter;
```
Comment (tarlepp) [inline src/Game/Formatter/HtmlGameFormatter.php:1]: PSR rules says that namespace is next line after <?php

File: src/Game/Game.php (hunk 2 above)
Comment (mannion007) [reply src/Game/Game.php:1]: I agree, never seen it on the same line before but I just stayed consistent with what was already there.

File: src/Game/GameFormatterInterface.php (hunk 1 above)
Comment (mannion007) [reply src/Game/GameFormatterInterface.php:9]: Good spot. PSR doesn't care and I'm inconsistent. I should configure my IDE better.

**Pull request closed without merging**